server/src/error.rs

```diff
@@ -291,5 +295,13 @@ impl From<serde_json::Error> for EdgeError {
     }
 }
 
+#[cfg(feature = "streaming")]
+impl From<SendError<Event>> for EdgeError {
+    // todo: create better enum representation. use this is placeholder
+    fn from(_value: SendError<Event>) -> Self {
+        EdgeError::TlsError
```

Comment:
Yes, I agree. Hence the comment. I just didn't want to start implementing a new error enum member when cleaning this up. We should handle this as part of testing, because I don't think it is the right errer.